hosting/how-to-host-on-sia.md

```diff
@@ -1,151 +1,239 @@
+Registry is a special form of storage representing a key-value store. It’s currently used heavily by Skynet. Hosts are paid more for storing registry entries than for storing regular data.
 
-You can go a step further and mitigate this issue by signing up for a Dynamic DNS \(DDNS\) service. If you have a static public IP address, you don't need to worry about this step. If you're not sure whether or not you have a static IP address, it won't hurt to set up DDNS anyway.
+Recommended registry size is 4 GB. However, you need to keep in mind that the registry is stored on your drive, and that used entries are loaded into RAM during the host startup. So, if you’re running low on RAM or drive space, you should consider reducing the registry size on your host.
 
-DDNS services work by assigning you a subdomain or URL, and having you run a small program or script on a device on your network - your Sia host computer will work fine. Some routers also have built-in DDNS support for certain DDNS services. When your public IP address changes, the DDNS client detects the change and updates your DDNS subdomain or URL with your new IP address. You can use your DDNS address to announce your Sia host, and renters will always be able to find you as long as you have DDNS set up and the IP update software running.
+Registry settings can only be changed in `siac` or using the Terminal in Sia-UI. For setting the registry size the command `host config registrysize <value>` is used, e.g.
+
+`host config registrysize 4GB`
+
+If you want to specify a different location for the registry file \(default is in your host folder\), you can use the command `host config customregistrypath <value>`, e.g.
+
+`host config customregistrypath “D:\\Sia”`
+
+### Proof Window
+
+When a storage contract ends, by default your host has about 24 h \(144 blocks\) to submit a storage proof, which is quite enough to allow for some downtime. If your host fails to submit a storage proof within this window, then only the part of the locked collateral is returned to your wallet, which is not risked, i.e., is not backing up any data uploaded to your host. Any revenue associated with your bandwidth usage is also paid to your wallet.
+
+This can only be changed in `siac`, or using the Terminal in Sia-UI with the command `host config windowsize <value>`, e.g.
+
+`host config windowsize 1d`
+
+## Announce Your Host
```

Comment:
This section seems out of place being so far from the DDNS/IP explanations

Reply:
Now that we have removed a few settings, it's not so far anymore. And I think it fits quite well in the flow. When you set up a host, you adjust the settings first and then announce, rarely the other way around.